lib/galaxy/jobs/runners/util/condor/__init__.py

```diff
@@ -35,87 +25,24 @@ def submission_params(prefix=SUBMIT_PARAM_PREFIX, **kwds):
     return submission_params
 
 
-def build_submit_description(executable, output, error, user_log, query_params):
+def build_submit_description(submit_params, query_params):
```

Comment:
Any chance you can write a second function to do this instead of replacing this one? It is used by Planemo also.